201-vmss-azureautomationdsc/webServer.ps1

```diff
@@ -0,0 +1,26 @@
+Configuration webServer
+{
+    Import-DSCResource -ModuleName PSDesiredStateConfiguration, xNetworking
+	
+	Node localhost
```

Comment:
just a note that this change has also been updated in corresponding parameters that point to the configuration name